components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -2159,6 +2159,9 @@ type Identity struct {
 	PrimaryEmail string   `json:"primaryEmail,omitempty"`
 	Readonly     bool     `json:"readonly,omitempty"`
 	Tokens       []*Token `json:"tokens,omitempty"`
+
+	// The last time this entry was touched during a signin.
+	LastSigninTime time.Time `json:"last_signin_time,omitempty"`
```

Comment:
**Question:** Is `lastSigninTime` the right property for us to rely on? That feels quite implicit. Should we attach some other more descriptive property to the user object, such as being "org-owned" user, etc? 🤔

Reply:
Yes it is implicit.
Also, the user is to be marked as org-owned, which is already happening. If `User.organizationId` is set, that account is considered org-owned.

Indeed, it would be great to persist some of those details explicitly. Thinking out loud about the shape: 

```
interface User {
  // ...
  emails: {
    email: string;
    updatedAt: time;
    provider?: string; // github.com, idp.comp.org
    verified: boolean;
    organizationId: uuid;
    personal: boolean;
    // ...
  }[]
}
```